packages/harmony/src/components/completion-check/CompletionCheck.module.css

```diff
@@ -0,0 +1,40 @@
+.container {
+  display: inline-flex;
+  align-items: center;
+  position: relative;
+  width: 16px;
+  height: 16px;
```

Comment:
try the `<Flex>` component!

Reply:
Ooh ty

Reply:
oh darn it doesn't do inline flex, but I can add a classname I think